src/main/java/tutoraid/model/lesson/Students.java

```diff
@@ -0,0 +1,51 @@
+package tutoraid.model.lesson;
+
+import static java.util.Objects.requireNonNull;
+import static tutoraid.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Lesson's students in TutorAid.
+ * Guarantees: immutable; is valid as declared in {@link #isValidStudents(String)}
+ */
+public class Students {
```

Comment:
How are we representing students? Looks like it's using a String here. Would some sort of collection or list be better?

Reply:
I was referring to #78 for this. I will update this class and the respective usages associated with this when the representation for Students has been changed there, since there may be different ideas on how students can be represented.